src/main/java/com/offz/spigot/staminaclimb/StaminaClimb.java

```diff
@@ -1,20 +1,36 @@
 package com.offz.spigot.staminaclimb;
 
 import com.offz.spigot.staminaclimb.Climbing.ClimbBehaviour;
+import com.offz.spigot.staminaclimb.ClimbingAxe.ClimbingAxeProperties;
 import com.offz.spigot.staminaclimb.Stamina.StaminaBar;
 import com.offz.spigot.staminaclimb.Stamina.StaminaTask;
 import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
 import org.bukkit.boss.BossBar;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.plugin.java.JavaPlugin;
 
 import java.util.UUID;
 
 public final class StaminaClimb extends JavaPlugin {
     @Override
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if(command.getName().equalsIgnoreCase("climbkit")) { //give the player the climbing axe
```

Comment:
I'd rather let players craft it using geary, instead of making it a kit. Derongan would know about that.